scipy/stats/tests/test_discrete_distns.py

```diff
@@ -101,6 +102,79 @@ def test_betabinom_bernoulli():
     assert_almost_equal(p, expected)
 
 
+def test_issue_10317():
+    alpha, n, p = 0.9, 10, 1
+    assert_equal(nbinom.interval(alpha=alpha, n=n, p=p), (0, 0))
+
+
+def test_issue_11134():
+    alpha, n, p = 0.95, 10, 0
+    assert_equal(binom.interval(alpha=alpha, n=n, p=p), (0, 0))
+
+
+def test_issue_7406():
+    np.random.seed(0)
+    assert_equal(binom.ppf(np.random.rand(10), 0, 0.5), 0)
+
+    # Also check that endpoints (q=0, q=1) are correct
+    assert_equal(binom.ppf(0, 0, 0.5), -1)
+    assert_equal(binom.ppf(1, 0, 0.5), 0)
+
+
+def test_issue_5122():
```

Comment:
This test does demonstrate that most, if not all, of gh-5122 is resolved. Since `qbinom(1, 10, 0)` in R agrees with Boost and existing SciPy behavior, I tend to think this was never a problem. If that's true, this PR closes gh-5122, too.